jsapp/scss/components/_kobo.table.scss

```diff
@@ -21,217 +26,329 @@
     }
 
-    .rt-td.rt-status {
-      line-height: inherit;
+    .rt-resizable-header-content {
       overflow: visible;
+      text-overflow: inherit;
 
-      .kobo-select__control {
-        background: transparent;
-        height: 30px;
-        min-height: 30px;
-        border: none;
-        border-radius: 0px;
-        box-shadow: none;
-        cursor: pointer;
+      .column-header-title,
+      .column-header-hxl-tags {
+        display: block;
+        width: 100%;
+        overflow: hidden;
+        text-overflow: ellipsis;
+      }
 
-        &:hover {
-          box-shadow: none;
-        }
+      .column-header-hxl-tags {
+        color: $kobo-gray-40;
       }
     }
+  }
 
-    .rt-link {
-      overflow: visible;
+  .rt-tr {
+    align-items: stretch;
+    justify-content: stretch;
+  }
 
-      .table-link {
-        cursor: pointer;
-        display: inline-block;
-        min-height: 28px;
+  .rt-tr-group {
+    max-height: $data-table-row-height;
+  }
 
-        &:hover {
-          color: $kobo-blue;
-        }
+  .rt-th, .rt-td {
+    padding: 5px 8px;
+    border-right: $data-table-border;
+    line-height: $data-table-row-height - (2 * 5px);
+    height: $data-table-row-height;
+  }
 
-        i {
-          font-size: 24px;
-          margin: 0 3px;
-          vertical-align: middle;
-        }
-      }
-    }
+  .rt-td.rt-numerical-value {
+    font-family: $font_mono;
+  }
 
-    .-pagination {
-      box-shadow: none;
-      border-top: none;
+  .rt-thead.-filters .rt-th.rt-status {
+    line-height: inherit;
+    overflow: visible;
+  }
 
-      .-center {
-        flex: 2;
-      }
+  .rt-td.rt-status {
+    // needed for dropdown
+    overflow: visible;
+  }
 
-      .-btn {
-        @extend .mdl-button, .mdl-button--colored;
+  .rt-sub-actions {
+    // needed for tooltips
+    overflow: visible;
+  }
 
-        &:hover {
-          background: transparent;
-        }
-      }
+  .table-submission-actions {
+    display: flex;
+    flex-direction: row;
+    align-items: center;
+    justify-content: space-around;
+
+    .checkbox {
+      margin-left: 5px;
+      margin-right: 5px;
     }
 
-    .pagination-bottom {
-      margin-top: 5px;
-      margin-bottom: 5px;
+    .checkbox__input {
+      display: block;
     }
 
-    .-loading {
-      background: $kobo-gray-96;
+    .table-link {
+      cursor: pointer;
+      min-height: 28px;
+      background: transparent;
+      border: 0;
+      padding: 0;
 
-      .-loading-inner {
-        display: block;
-        text-align: center;
-        font-size: 18px;
-        opacity: 0.8;
-        color: $kobo-gray-40;
+      &:hover {
+        color: $kobo-blue;
+      }
 
-        i {
-          margin-right: 12px;
-          vertical-align: middle;
-          font-size: 24px;
-        }
+      i {
+        font-size: 24px;
+        margin: 0 3px;
+        vertical-align: middle;
       }
     }
+  }
+
+  .-pagination {
```

Comment:
Curious why the prepended `-` notation?

Reply:
This is a classname from `react-table` and I have no idea why they chose it :D Probably they went with `-pagination` as `pagination` would definitely be a popular class name and conflicts would happen.